docs/installation/advanced_topics.asciidoc

```diff
@@ -107,6 +134,203 @@ View of the opened security events:
 
 In the example above, you can see that the security_event table is _left joined_ to the class, node and locationlog tables. Using that strategy we make sure all the security events are listed even on deleted nodes. Then, base conditions are added to filter out outdated locationlog entries as well as include devices without locationlog entries. Removing those conditions would lead to duplicate entries being shown since the report would reflect all the historical locationlog entries.
 
+==== SQL
+
+When `type=sql` PacketFence uses MySQL script/batch mode to manually build the SQL query including the execution of multiple statements. This provides complete query control as well as the ability to manage the SQL session and the SQL transaction. This is the preferred mode where SQL optimization is needed to execute complex queries, or for those more comfortable with raw (non-abstract) SQL.
+
+  sql=SELECT * FROM sponsors;
+
+Multiline block (heredoc) is required when executing multiple statements. Each statement should be terminated with a semi-color ";".
+
+NOTE: SQL execution exits on the first error and returns the result set of the last successful statement.
+
+The following attributes are available when using `type=sql`: [options="compact"]
```

Comment:
`[options="compact"]` is an asciidoc list setting; move it on its own line, right before the list definition.